worker/README.md

```
@@ -1,394 +1,99 @@
-- Grafana Cloud integration
-- Pay-as-you-go pricing
-
-**Getting Credentials:**
-1. Login to [Grafana Cloud](https://grafana.com)
-2. Go to your stack → Details
-3. Copy Instance ID and generate API Key with MetricsPublisher permission
-4. Copy your region from the Prometheus endpoint URL
-
-**See:** [DEPLOYMENT_GUIDE.md](../DEPLOYMENT_GUIDE.md#option-3-grafana-cloud) for complete setup instructions
+1. **Install Dependencies**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)